server.go

```diff
@@ -142,7 +146,47 @@ func handleProcessWebhook(c echo.Context) (err error) {
 	case webhook.PullRequestPayload:
 		switch payload.Action {
 		case "opened", "reopened", "synchronize":
-			res, err := ourGithub.HandlePullRequest(c.Logger(), postgresDB, payload, appId, getCurrentCLAVersion())
+			// Getting a JWT Apps Transport to ask GitHub about stuff that needs a JWT for asking, such as installInfo
+			atr, err := ghinstallation.NewAppsTransportKeyFromFile(http.DefaultTransport, int64(appId), filenameTheClaPem)
+			if err != nil {
+				c.Logger().Error(err)
+				return c.String(http.StatusBadRequest, err.Error())
+			}
+
+			ghJWTClient := ourGithub.NewJWTClient(&http.Client{Transport: atr}, c.Logger(), int(payload.Installation.ID))
+
+			installInfo, err := ghJWTClient.GetInstallInfo()
+			if err != nil {
+				c.Logger().Error(err)
+				return c.String(http.StatusBadRequest, err.Error())
+			}
+
+			// See if we can move this to a longer lived thing, maybe? It's going to recreate the transport and http Client each time we get a payload
```

Comment:
I have ideas on how we can do this. INSTALL ID may change intermittently, but most other things should stay the same. We can allow someone to update the installID on ghClient, and inside we can say "hey, before you make a request, refresh your token real quick". That's my quick thought on it, so that ghClient can be longer lived?